src/renderer/dx/DxRenderer.cpp

```diff
@@ -653,13 +657,13 @@ void DxEngine::_ReleaseDeviceResources() noexcept
 }
 
 [[nodiscard]] HRESULT DxEngine::SetWindowSize(const SIZE Pixels) noexcept
+try
 {
     _sizeTarget = Pixels;
-
-    RETURN_IF_FAILED(InvalidateAll());
-
+    _invalidMap.resize(_sizeTarget / _glyphCell, true);

```

Comment:
is `_glyphCell` scaled for DPI?

Reply:
I'll look into that. Might be related to Dustin's issues with High DPI.

Reply:
`_glyphCell` is not scaled for DPI.